cloudbuild.yaml

```diff
@@ -119,19 +119,30 @@ steps:
 #
 
 - name: "make-docker"
-  id: build
+  id: build-images
   waitFor:
     - lint
-    - ensure-build-sdk-image-base
   dir: "build"
-  args: [ "-j", "4", "build"]
+  args: [ "-j", "4", "build-images"]
 - name: "make-docker"
   id: push-images
   waitFor:
-    - build
+    - build-images
   dir: "build"
   args: [ "-j", "3", "push"]
 
+#
+# Build all the SDKs
+#
+
+- name: "make-docker"
+  id: build-sdks
+  waitFor:
+    - lint
+    - ensure-build-sdk-image-base
+  dir: "build"
+  args: [ "-j", "4", "--output-sync=target", "build-sdks"]
```

Comment:
It might be worth adding a note to https://github.com/googleforgames/agones/tree/master/build/build-sdk-images#adding-support-for-a-new-language to change the parallelism argument here when adding support for a new language.

Reply:
Admittedly, only C++ has a build command, so -j 4 is currently overkill anyway.

Reply:
I should be clearer on this - there is currently only 1 sdk that is building, but I set it to 4 to (a) be future proof and (b) since we have multiple concurrent steps, I wanted to max things out at half the cores on the machine.

So I don't _think_ we need a comment on this, at least at this time.

Does that sound reasonable?

Reply:
yep.